commit 6584682c53ef5002b2cd8b24085b704c0ec69697
Author: Abdellatif Ait boudad <a.aitboudad@gmail.com>
Date:   Fri Nov 10 17:38:37 2017 +0000

    patch

diff --git a/ngc.js b/ngc.js
index cb48466..2f62191 100644
--- a/ngc.js
+++ b/ngc.js
@@ -21,6 +21,7 @@ function prepareTsConfig(ngPkg, outFile) {
         compilerOptions.flatModuleId = ngPkg.fullPackageName;
         compilerOptions.flatModuleOutFile = `${ngPkg.flatModuleFileName}.js`;
         tsConfig['files'] = [ngPkg.entryFile];
+        tsConfig['compilerOptions']['paths'] = { "@ngx-formly/core": ["../../../../../dist/core"] };
         yield fs_extra_1.writeJson(outFile, tsConfig);
     });
 }
@@ -42,4 +43,4 @@ function ngc(ngPkg, basePath) {
     });
 }
 exports.ngc = ngc;
-//# sourceMappingURL=ngc.js.map
\ No newline at end of file
+//# sourceMappingURL=ngc.js.map
