commit 2240dcd6e2fd2de23e5918bbfb64ca51a50d37ae
Author: Abdellatif Ait boudad <a.aitboudad@gmail.com>
Date:   Sun Mar 25 12:09:50 2018 +0100

    www

diff --git a/compile-ngc.transform.js b/compile-ngc.transform.js
index a8d2c51..451ea76 100644
--- a/compile-ngc.transform.js
+++ b/compile-ngc.transform.js
@@ -20,6 +20,7 @@ exports.compileNgcTransform = transform_1.transformFromPromise((graph) => __awai
     const tsConfig = entryPoint.data.tsConfig;
     // Add paths mappings for dependencies
     const entryPointDeps = entryPoint.filter(nodes_1.isEntryPoint);
+    tsConfig.options.paths = { "@ngx-formly/core": ["../../dist/core"] };
     if (entryPointDeps.length > 0) {
         if (!tsConfig.options.paths) {
             tsConfig.options.paths = {};
@@ -42,4 +43,4 @@ exports.compileNgcTransform = transform_1.transformFromPromise((graph) => __awai
     entryPoint.data.metadata = compilationResult.metadata;
     return graph;
 }));
-//# sourceMappingURL=compile-ngc.transform.js.map
\ No newline at end of file
+//# sourceMappingURL=compile-ngc.transform.js.map
