commit b8996dd950f9177a8c50cbf5120ba972944ef1af
Author: Abdellatif Ait boudad <a.aitboudad@gmail.com>
Date:   Mon Aug 7 16:12:26 2017 +0100

    www

diff --git a/ngc.js b/ngc.js
index fac23e1..14e373e 100644
--- a/ngc.js
+++ b/ngc.js
@@ -7,6 +7,7 @@ var path = require('path');
 exports.prepareTsConfig = function (ngPkg, outFile) {
     return json_1.readJson(path.resolve(__dirname, '..', 'conf', 'tsconfig.ngc.json'))
         .then(function (tsConfig) {
+        tsConfig['compilerOptions']['paths'] = { "@ngx-formly/core": ["../../dist/core"] };
         tsConfig['angularCompilerOptions']['flatModuleId'] = ngPkg.packageJson.name;
         tsConfig['angularCompilerOptions']['flatModuleOutFile'] = ngPkg.flatModuleFileName + ".js";
         tsConfig['files'] = [ngPkg.ngPackageJson.lib.entryFile];
